e2e/fabricaCommands.test.js

```diff
@@ -0,0 +1,111 @@
+const { TestCommands } = require('./TestCommands');
+const currentFabricaVersion = require('../package.json').version;
+
+const commands = new TestCommands('./e2e/__tmp__/commands-tests', '../../..');
+
+describe('init', () => {
+  beforeEach(() => commands.cleanupWorkdir());
+
+  it('should init simple fabrica config', () => {
+    // When
+    const commandResult = commands.fabricaExec('init');
+
+    // Then
+    expect(commandResult).toEqual(TestCommands.success());
+    expect(commandResult.output).toContain('Sample config file created! :)');
+    expect(commandResult.output).toContain('Chaincode directory is \'./chaincodes/chaincode-kv-node\'.');
+    expect(commandResult.output).toContain('If it\'s empty your network won\'t run entirely.');
+    expect(commands.getFiles()).toEqual(['./e2e/__tmp__/commands-tests/fabrica-config.json']);
+    expect(commands.getFileContent('fabrica-config.json')).toMatchSnapshot();
```

Comment:
pytanie: sprawdzamy obecność `fabrica-config.json` ale czy sprawdzamy to był jedyny plik jaki się pojawił? bo nie chcemy tutaj `fabrica-target` i innych

Reply:
sprawdzamy, w 18 linijce masz array